config/initializers/ruby_money.rb

```diff
@@ -1,8 +1,28 @@
 # frozen_string_literal: true
 
+require 'money/bank/currencylayer_bank'
+
 Money.locale_backend = :i18n
 
-eu_bank = EuCentralBank.new
-Money.default_bank = eu_bank
+if Rails.env.test? || Rails.env.development?
```

Comment:
There is `Rails.env.local?` which is exactly an equivalent of "test or dev"

Reply:
Oh that's nice, updated.